how-to/containerd-kata.md

```diff
@@ -0,0 +1,357 @@
+is implemented in Kata Containers v1.5.0.
+
+## Install
+
+### Install Kata Containers
+
+Follow the instructions to [install Kata Containers](https://github.com/kata-containers/documentation/blob/master/install/README.md).
+
+### Install containerd with cri plugin
+
+> **Note:** `cri` is a native plugin of containerd 1.1 and above. It is built into containerd and enabled by default. 
+> You don't need to install `cri` if you have containerd 1.1 or above. Just check if the `cri` plugin has been disabled 
+> in the containerd configuration file.
+
+Follow the instructions from the [CRI installation guide](http://github.com/containerd/cri/blob/master/docs/installation.md).
+
+And check if `containerd` is now available:
+  
+```bash
+$ command -v containerd
+```
+
+### Install CNI plugins
+
+> **Note:** You don't need install CNI plugins if you don't want to use containerd with Kubernetes.
+> And if you have installed Kubernetes with `kubeadm`, you may have already had CNI plugins installed.
+
+You may manually install CNI plugins as follows:
+
+```bash
+$ go get github.com/containernetworking/plugins
+$ pushd $GOPATH/src/github.com/containernetworking/plugins
+$ ./build_linux.sh
+$ sudo mkdir /opt/cni
+$ sudo cp -r bin /opt/cni/
+$ popd
+```
+
+### Install cri-tools
+
+> **Note:** `cri-tools` is a set of tools for CRI for the development and testing. For users only want to use containerd
+> with kubernetes, you may skip the `cri-tools`.
+
+You may install the `cri-tools` from source code:
+
+```bash
+$ go get github.com/kubernetes-incubator/cri-tools
+$ pushd $GOPATH/src/github.com/kubernetes-incubator/cri-tools
+$ make
+$ sudo -E make install
+$ popd
+```
+
+## Configuration
+
+### Configure containerd to use Kata Containers
+
+By default, the configuration of containerd locates at `/etc/containerd/config.toml`, and the 
+`cri` plugins are placed in the following section:
+
+```toml
+[plugins]
+  [plugins.cri]
+    [plugins.cri.containerd]
+      [plugins.cri.containerd.default_runtime]
+        #runtime_type = "io.containerd.runtime.v1.linux"
+
+    [plugins.cri.cni]
+      # conf_dir is the directory in which the admin places a CNI conf.
+      conf_dir = "/etc/cni/net.d"
+```
+
+Let's add Kata Containers to the configurations.
+
+#### Kata Containers as a RuntimeClass
+
+For 
+- Kata Containers v1.5.0 or above (including 1.5.0-rc)
+- Containerd v1.2.0 or above
+- Kubernetes v1.12.0 or above
+
+The  RuntimeClass is suggested.
+
+The following configuration include three runtime classes:
+- `plugins.cri.containerd.runtimes.runc`: the runC, and it is the default runtime;
+- `plugins.cri.containerd.runtimes.kata`: it is magic in containerd (ref [the doc here](https://github.com/containerd/containerd/tree/master/runtime/v2#binary-naming)), in which the dot connected string `io.containerd.kata.v2` will be translated to `containerd-shim-kata-v2`, i.e. the binary name of the kata implementation of shim v2 API;
```

Comment:
Lines 151-152 suggested rewrite (I very easily could have totally changed meaning here - hopefully not): 
- `plugins.cri.containerd.runtimes.kata`: The function in containerd (reference [the document here](https://github.com/containerd/containerd/tree/master/runtime/v2#binary-naming)) where the dot-connected string `io.containerd.kata.v2` is translated to `containerd-shim-kata-v2` (i.e. the binary name of the kata implementation of shim v2 API), [Containerd Runtime V2 (Shim API)](https://github.com/containerd/containerd/tree/master/runtime/v2).